libraries/WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -53,6 +59,13 @@ class FunctionRequestHandler : public RequestHandler {
             _ufn();
     }
 
+    void raw(WebServer& server, String requestUri, HTTPRaw& raw) override {
+        (void)server;
+        (void)raw;
+        if (canRaw(requestUri))
+            _ufn();
```

Comment:
```suggestion
        if (canRaw(requestUri)) {
            _ufn();
        }
```

Reply:
Why ? The rest of the code does not have this formatting.

Reply:
It's better to avoid bracketless ifs. The rest of the code is old and shouldn't be like this. Either way all the code base will be reformatted soon and all short ifs will be removed.